tests/test_chat_typing.py

```
@@ -0,0 +1,182 @@
+        r = await client.post(
+            f"/api/chat/channels/{ch_id}/typing",
+            json={"author_id": "user"},
+        )
+        assert r.status_code == 200
+        listing = app.state.typing.list(ch_id)
+        assert "user" in listing["human"]
+
+
+@pytest.mark.asyncio
+async def test_post_thinking_start_marks_registry(tmp_path):
+    app, client, token = await _setup_client(tmp_path)
```

Ruff (0.15.10)

[warning] Unpacked variable `token` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)